chaincode/ledger_db.go

```diff
@@ -529,6 +529,13 @@ func (db *LedgerDB) AddComputePlanEvent(ComputePlanID, status string) error {
 		return err
 	}
 	cp.AlgoKeys = algokeys
+
+	modelHashes, err := db.GetIndexKeys("model~computePlanID~hash", []string{"model", ComputePlanID})
+	if err != nil {
+		return err
+	}
+	cp.ModelHashes = modelHashes
```

Comment:
for the largest compute plans I think we will grow constantly. Maybe we should consider sending only the ones that still to be removed or use another limitation.

Reply:
You right here. It will grow and reach a critical size.
We can go back to the first design suggested for intermediary models: removing at each step the ones that were already used. It will require a bit more logic when handling the compute plan event (we don't want to "clean" an ongoing compute plan).
Or we can only send N models in the event and make the cleaning task call a specific end point to retrieve the next M models to delete. What do you think @inalgnu, @Kelvin-M @AurelienGasser ?